src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mans = people.filter(male =>
+    male.sex === 'm'
+    && (century > 0 ? Math.ceil(male.died / 100) === century : true)
```

Comment:
```suggestion
    male.sex === 'm'
    && (
      century > 0
        ? Math.ceil(male.died / 100) === century 
        : true
    )
```

Reply:
you didn't fix this comment Ihor wrote to you